extensions/ricescore/src/index.tsx

```diff
@@ -0,0 +1,95 @@
+import { Form, ActionPanel, CopyToClipboardAction } from "@raycast/api";
+import { markdownTable } from "markdown-table";
+import { useState } from "react";
+
+function parseNumberText(text: string | undefined): number {
+  let t = text || "";
+  if (t.includes(",")) {
+    t = t.replace(",", ".");
+  }
+  return parseFloat(t);
+}
+
+function getMarkdown(
+  reach: string | undefined,
+  impact: string | undefined,
+  confidence: string | undefined,
+  effort: string | undefined,
+  score: string
+): string {
+  const getOrDefault = (num: string | undefined): string => {
+    if (num) {
+      return num;
+    } else {
+      return " ";
+    }
+  };
+  getOrDefault(reach);
+  return markdownTable(
+    [
+      ["Reach", "Impact", "Confidence", "Effort", "RICE score"],
+      [getOrDefault(reach), getOrDefault(impact), getOrDefault(confidence), getOrDefault(effort), score],
+    ],
+    { align: ["c", "c", "c", "c", "c"] }
+  );
+}
+
+export default function Command() {
+  const [reach, setReach] = useState<string>();
+  const [impact, setImpact] = useState<string>();
+  const [confidence, setConfidence] = useState<string>();
+  const [effort, setEffort] = useState<string>();
+
+  const reachNum = parseNumberText(reach);
+  const impactNum = parseNumberText(impact);
+  const confidenceNum = parseNumberText(confidence);
+  const effortNum = parseNumberText(effort);
+  const score = (reachNum * impactNum * (confidenceNum / 100.0)) / effortNum;
+  const roundScore = Math.round(score * 100.0) / 100.0;
+  const isValid = !Number.isNaN(roundScore);
+  const scoreText = isValid ? roundScore.toString() : "";
+
+  const md = getMarkdown(reach, impact, confidence, effort, scoreText);
+
+  return (
+    <Form
+      actions={
+        <ActionPanel>
+          {isValid && <CopyToClipboardAction title="Copy RICE score to Clipboard" content={scoreText} />}
+          {isValid && <CopyToClipboardAction title="Copy RICE score markdown to Clipboard" content={md} />}
```

Comment:
Actions must also follow title case convention.

Reply:
Also consider shortening the title, it doesn't seem to fit in action panel. Since you already in "RICE Score" extension, you can omit "RICE" in the action. "Copy Score" and "Copy Score Markdown" would do the job.

Reply:
New version

![CleanShot 2021-10-30 at 12 27 44@2x](https://user-images.githubusercontent.com/3163807/139529255-a8f2dc11-b4d2-47f3-b6b5-781cbd617502.png)